src/test/java/com/oltpbenchmark/benchmarks/twitter/TestTwitterLoader.java

```diff
@@ -27,7 +27,6 @@
     @Override
     protected void setUp() throws Exception {
         super.setUp(TwitterBenchmark.class, IGNORED_TABLES, TestTwitterBenchmark.PROC_CLASSES);
-        this.workConf.setScaleFactor(0.1);
```

Comment:
I see this change with other test loaders, does it not cause the scalefactor to be too large?

Reply:
so this is addressed by setting a default ScaleFactor in the parent AbstractTestCase instead of relying on individual test loaders to think of it.  see https://github.com/cmu-db/benchbase/blob/76917cf2812c02043cf09458931af0c2f70ccf4b/src/test/java/com/oltpbenchmark/api/AbstractTestCase.java#L52

Reply:
so short answer is no.  all test cases will use the smaller scale factor